pages/resources/insights.mdx

```diff
@@ -0,0 +1,185 @@
+| joins       | integer      | The number of users who joined the server using this invite link |
+
+<RouteHeader method="GET" url="/guilds/{guild.id}/analytics/growth-activation/joins-by-referrer">
+  Get Guild Insights by Joins by Referrer
+</RouteHeader>
+
+###### Query String Params
+
+| Field    | Type         | Description                                            |
+| -------- | ------------ | ------------------------------------------------------ |
+| start    | ISO8601 date | The start date for the insights data                   |
+| end      | ISO8601 date | The end date for the insights data                     |
+| interval | integer      | [The interval for the insights data](#interval-values) |
+
+###### Response Body
+
+| Field                | Type         | Description                                                        |
+| -------------------- | ------------ | ------------------------------------------------------------------ |
+| day_pt               | ISO8601 date | The date the referral was made                                     |
+| referring_domain ^1^ | string       | The referring domain that led to the referral link being clicked   |
+| joins                | integer      | The number of users who joined the server using this referral link |
+
+^1^ The `referring_domain` field represents what the `Referer` header contained when they joined the server.
+
+<RouteHeader method="GET" url="/guilds/{guild.id}/analytics/growth-activation/joins-by-source">
+  Get Guild Insights by Joins by Sources
+</RouteHeader>
+
+###### Query String Params
+
+| Field    | Type         | Description                                            |
+| -------- | ------------ | ------------------------------------------------------ |
+| start    | ISO8601 date | The start date for the insights data                   |
+| end      | ISO8601 date | The end date for the insights data                     |
+| interval | integer      | [The interval for the insights data](#interval-values) |
+
+###### Response Body
+
+| Field             | Type         | Description                                                               |
+| ----------------- | ------------ | ------------------------------------------------------------------------- |
+| day_pt            | ISO8601 date | The date the source was recorded                                          |
+| discovery_joins   | integer      | The number of users who joined the server through the discovery hub       |
+| invites           | integer      | The number of users who joined the server through invite links            |
+| vanity_joins      | integer      | The number of users who joined the server through the server's vanity URL |
+| hubs_joins        | integer      | The number of users who joined the server through the server's hubs       |
+| bot_joins         | integer      | The number of bots that joined the server                                 |
```

Comment:
this is incorrect, don't just guess
it's the number of joins through bot means (i.e. guilds.join, see the join source type enum in guild.mdx)
same applies to below field